frontend/src/components/common/Link.tsx

```diff
@@ -23,6 +27,7 @@ export interface LinkProps extends LinkBaseProps {
   state?: {
     [prop: string]: any;
   };
+  drawerEnabled?: boolean;
```

Comment:
this prop is not used, unless I'm missing something

Reply:
done